### ISSUE TITLE
Thank you + a quick question

### ISSUE THREAD

**Issue by tkellen** [andyshinn/kuberdoo#2]
First, thank you for sharing this project, it's awesome :dancer: 

I am about to attempt a build of my own. The README for this project mentions you have updated CAD files... and it looks like you made an update at OnShape over the weekend? Is that the v4 model you mentioned? If not, would you be willing to share? I'd love to give it a shot and would be more than happy to share photos/a report of how it went.

![panels](https://user-images.githubusercontent.com/1004324/39718329-f5edac08-5203-11e8-939a-b947760a54bd.gif)

Cheers!

**Comment by andyshinn** [comment]
Thanks!

I'm glad you asked first as I haven't made all the changes yet. Those are only some of the v4 fixes. There are still two constraints that need to be fixed.

1. The more major one is the slot alignment of the back panel. If you check the constraints you will see that the notches for the top and bottom panel don't quite line up with the ones in the back panel (0.5mm off) which requires some slight trimming with snips so they fit together.
1. I didn't fix yet is the hole alignment for the Netgear switch. One of the mounting holes is a couple mm off and you can only use 3 of the holes.

I'll try and go through the drawing again this week and see if I can fix these. The changes already made are more minor (just giving a little bit better spacing and room for some parts).

I'll close this issue once I tag a v4 version with those fixes. I've also got a bunch of other documentation stuff locally that I haven't pushed yet which should help with instructions for building.

**Comment by tkellen** [comment]
Neat, thanks so much for the reply @andyshinn. I'll hang tight.

One additional thought just occurred to me. In your README you recommend a newer PSU, (which I did order). Here's the spec sheet on it:
![lrs100](https://user-images.githubusercontent.com/1004324/39723376-a6c5be12-5213-11e8-8948-09f65999c1c5.png).

Not suggesting you need to update it for me, but that seems like an easy wrinkle to miss if you don't specify which the V4 is designed for (thinking about the mounting holes).

Also, as a total aside, if you email me your address (tyler@sleekcode.net) I'll order two cases and send the second your way!

**Comment by andyshinn** [comment]
Ah, you are correct. The mounting is different than the RS-100. But it shouldn't be too difficult to change. I should be able to change it for that hole mounting when I take a look at it.

Also, there is something else about the PSU that I just remembered. The depth holes is rated for 2mm and I think I have 6mm thumb screws on the list of parts for mounting this. The 3mm thickness of the case means that the screws are technically too long for the PSU mounting. Though, I am using 6mm in mine and it seems to be fine (doesn't seem to be hitting anything on the PSU board but I'm guessing the mac depth is so that it doesn't come in contact with the circuit board).

**Comment by tkellen** [comment]
Good to know! I'll keep that in mind when I'm mounting the PSU and have a rotary tool handy to cut the thumbscrew short if needed.

**Comment by andyshinn** [comment]
I made the edits and tagged it as V4 version in OnShape. I also fixed an erroneous deletion of the assembly panel tab as well (so you can view the assembled case again).

Take a look and let me know if anything looks strange. I looked it over briefly but I think it looks good. The edits were pretty minor. I've just added two holes for the LRS-100 so either could be used. It should sit in the same position from the bottom terminal edge of the PSU.

Also, I may be wrong on the screw depth for the LRS-100. The drawing actually says L=4.5 (I'm guessing this is the thread depth) so deeper than I thought!

**Comment by tkellen** [comment]
I haven't touched a CAD program in... I guess about 6 years?
https://github.com/tkellen/super-portable-alumium-oven

To my untrained eye, this looks good @andyshinn. Thank you so much! 

I'm going to upload these to pololu this afternoon. Say, you didn't reply RE: my having a second case made and sending it your way. Would you like me to do that or no?

**Comment by zarez** [comment]
One note: C14 power connector. Depends on where you source it, you might have to sand the case opening a bit. Dont ask me how I know.

**Comment by andyshinn** [comment]
Thanks for the info zarez.

Thanks for the case offer. But no need to send me one :)

**Comment by zarez** [comment]
I was thinking adding some ssd that I have laying around, but found that the port is blocked by psu. I guess I can get the boards upside down but have no time to take apart now ( I am actually using this puppy alot)

**Comment by zarez** [comment]
@andyshinn I was actually thinking about asking UDOO guys if they are interested in making strip down version of the board for this kind of usage. I think will little bit of code you can make you onw mini cloud

**Comment by andyshinn** [comment]
Cool. If you have some suggestions for a next generation version of the case I would start a new issue instead of adding to this one. Let's keep this issue specific to the V4 case cutout.

If you checkout the UDDO forums there is a case section and there are some other mini cloud ideas there as well. Their stripped down boards are all ARM based at the moment. If I had to guess, it is likely that the Atom processor architecture is what limits what they can strip out of the x86 board.

**Comment by tkellen** [comment]
Most of the parts arrived today! I'm heading out for the evening but I wanted to post a quick update for anyone thinking of ordering the v4. Hold off a bit.

![img_20180511_170906705](https://user-images.githubusercontent.com/1004324/39947071-98e586c8-553e-11e8-8caf-7ce4d95f09ea.jpg)

Haven't put my head to what happened yet, but unless I've assembled something wrong, 
I think the vent spacing for the boards needs another pass. Good call not ordering two @andyshinn :smile:

More this weekend when I can look more closely!

**Comment by tkellen** [comment]
Back again! So, the vent spacing is perfect now, but the top six vents need to come down ~11mm. Unfortunately the battery in my calipers is dead, but here's a shot showing that:
![image](https://user-images.githubusercontent.com/1004324/39960891-00288f64-55f9-11e8-8a96-530099ef92e9.png)

The bottom vent needs to remain where it is, be moved down about half that distance, or be removed. Also, I think it would be ideal if the power/reset switches were centered vertically with the vents.

I am going to sign up for OnShape now and try to sort out how to make these adjustments and share them back.

Other wrinkles:

1. The PSU hole spacing doesn't appear to be correct for the LRS-100 (photo of this to follow later).
2. The C14 receptacle cutout needs to be about 1mm taller.

Anything you'd like to see that I haven't mentioned @andyshinn?

**Comment by tkellen** [comment]
Not quite sure how to share these back with you @andyshinn, but, I reduced the vent count to 6, set the constraint for distance of first vent to 31.6 from the bottom and adjusted the distance for the power/reset switches to start at 35.1 so they would be centered.

![adjustments](https://user-images.githubusercontent.com/1004324/39962538-9a8a1eb2-561f-11e8-998c-1d80c864d746.png)

The holes for the PSU turned out to be fine and a bit of shaving with my pocket knife got the C14 receptacle in place.

I'm curious to hear your thoughts about skipping a vent for the router? Seems like there will be plenty of airflow with this 140mm fan and all the other vents...

**Comment by tkellen** [comment]
...also extended the side vents to start and stop with the main vents. Here's a link:

https://cad.onshape.com/documents/8ba54d73704d0bbd7765ead9/v/0d3974aec15f72b5fb83378f/e/0956c2e4007c3a72cca42eb4

I am slowly backing away from this CAD program now. I can see myself blowing huge amounts of time in here tinkering :p

**Comment by andyshinn** [comment]
That OnShape document doesn't seem to be public (I can't open it).

I guess the switches could be aligned with the vents. They were originally but I left them in place when I fixed the alignment with the heatsinks.

What is the reasoning for removing the vent for the router?

**Comment by tkellen** [comment]
Whoops! Just made the document public. Reason for removing the router vent is based on a silly desire for the vent spacing to be symmetrical throughout and a belief (I am hoping you can validate) that it probably isn't needed.

**Comment by andyshinn** [comment]
There is actually a OnShape feature to make different configurations for a document. I'll probably use this feature to make configurations for the different power supplies and vent layouts. It should also make it easier to add different configurations later on for different parts / part sizes.

**Comment by hfoakley1948** [comment]
My parts have arrived and I would like to order a case at Pololu. Is there a v4 at OnShape or a 
version available at Pololu that I could reference for an order? Great project.

**Comment by andyshinn** [comment]
I did end up tagging a v4 version at OnShape you can reference (you may need to login to OnShape to see all the versions). But I can't remember if there were problems in the latest version.

@tkellen can you confirm if there were some changes that should be made / stuff not lining up correctly? From your previous message it sounded like maybe you had figured out everything and there were not any changes needed.

**Comment by tkellen** [comment]
@andyshinn, the correct spacing to the bottom of the first udoo board is shown here:
![adjustments](https://user-images.githubusercontent.com/1004324/39962538-9a8a1eb2-561f-11e8-998c-1d80c864d746.png)

(note that this drawing omits the vent for the router). All the other changes I mentioned are not strictly necessary.

**Comment by tkellen** [comment]
Also, just to confirm, https://cad.onshape.com/documents/8ba54d73704d0bbd7765ead9/v/0d3974aec15f72b5fb83378f/e/0956c2e4007c3a72cca42eb4 is exactly the same as your v4 with the exception to the changes I made for vent spacing and power switch alignment (which I can confirm worked out as planned). I think it would make sense to replace your front panel with the one from this project and call it v5 or some such.

**Comment by andyshinn** [comment]
Wouldn't that make the vents start at the bottom of the board? My original intent was for the vents to align above the board so that air flowed over the top of it and directly over the heatsink.

Also, I'm not quite sure why we need to omit vent for the router. Is that just because in this new configuration there isn't enough spacing for it?

@hfoakley1948 in either case, the v4 version should work. Any changes we make for a v5 version would be vent spacing / removal of the vent for the router. If those are not a concern then you could take the drawings for v4 now and send them to Pololu. I'm not sure if Pololu has a way to reference previous cuts.

**Comment by tkellen** [comment]
Sorry for the slow reply @andyshinn, I've been out of town / offline. I see what you mean now! I have the vents lined up with the bottom of the board so I can use the ports. I think doubling their height might serve both purposes? RE: omitting the vent for the router, that was based on desire for the vents to be evenly spaced across the entire front panel and a belief there would be enough airflow without it (so far an accurate one).

**Comment by andyshinn** [comment]
I see. I might play around with that. The original intent was only for airflow and not port access. The original spacing did line up with the top board ports but that was only by accident.

The slots could serve both purposes if they were maybe slightly wider and aligned at the bottom of the board.

**Comment by ritchington** [comment]
Hey Friends!

So in the spirit of ruining everything (I don't know how to use Onshape), I put Tkellen's v5 into Tinkercad and made some edits, then exported to SVG. I built this last year, but my cat knocked it off a shelf and broke the case, so I've decided to recut the case with some improvements...

- Nearly Doubled the front slot height, for access to USB Ports, and above board cooling
- Removed the front Vertical Ventilation slots (You'll see why in a second)
- Modified Side holes to mount TWO POWER SUPPLIES (LRS-75-5 + LRS-100-12) and still fit
- Removed otherwise superfluous holes
- Made the back power supply hole just the teeniest bit deeper
- Increased the size of the ethernet cable hole (I bought a mount that was bigger than original I think)
- Added Power Supply venting holes to the bottom and top
- Created a Tray piece, that mounts on the UdooX86 holes, with holes for Raspberry PI B, Odroid Xu4, and 2.5inch HDD/SSD

I'm too poor to buy 6 Udoo X86's, so now I can use it as a prototyping/homelab for a whole heap of things. 5V can now power PIs, Odroid, HDD, and any other crap you wanna put in there.

Files attached here :)
Enjoy!

[Kuberdoo Ritchington v6.zip](https://github.com/andyshinn/kuberdoo/files/3144497/Kuberdoo.Ritchington.v6.zip)
